apps/agreement/contracts/Agreement.sol

```diff
@@ -1127,9 +1127,7 @@ contract Agreement is IAgreement, AragonApp {
             return false;
         }
 
-        // TODO: update with new ACL version: no need to pass challenger address by parameter
-        bytes memory params = ConversionHelpers.dangerouslyCastUintArrayToBytes(arr(_challenger));
-        return currentKernel.hasPermission(_challenger, address(_disputable), CHALLENGE_ROLE, params);
+        return currentKernel.hasPermission(_challenger, address(_disputable), CHALLENGE_ROLE, new bytes(0));
     }
 
     /**
```

Comment:
I think we forgot the most important change here—updating the [`canPerform()`](https://github.com/aragon/aragon-apps/blob/80e4b5935fbac4bb2635e6c21f72ecc3169485be/apps/agreement/contracts/Agreement.sol#L662) function.

Reply:
True, I think the signature was updated before but the previous implementation was kept. Pushed a fix here https://github.com/aragon/aragon-apps/pull/1212

Reply:
True, I think the signature was updated before but the previous implementation was kept. Pushed a fix here https://github.com/aragon/aragon-apps/pull/1212